docs/setup.asciidoc

```diff
@@ -5,9 +5,17 @@ NOTE: For the best reading experience,
 please view this documentation at https://www.elastic.co/guide/en/apm/agent/nodejs/current/advanced-setup.html[elastic.co]
 endif::[]
 
-== Advanced Setup and Configuration
+== Setup and Configuration
+
+The following pages will help you set up and configure your APM Agent.
+
+* <<starting-the-agent>>
+* <<configuring-the-agent>>
+* <<agent-configuration-object>>
```

Comment:
This bullet point (and the next) is technically sub headlines on the "Configuring the agent" page. What do you think of indenting these two bullets one level?